pkg/operator/ceph/client/controller.go

```diff
@@ -176,7 +176,7 @@ func (r *ReconcileCephClient) reconcile(request reconcile.Request) (reconcile.Re
 	}
 
 	// Populate clusterInfo during each reconcile
-	r.clusterInfo, _, _, err = opcontroller.LoadClusterInfo(r.context, r.opManagerContext, request.NamespacedName.Namespace)
+	r.clusterInfo, _, _, err = opcontroller.LoadClusterInfo(&cephCluster.Spec, r.context, r.opManagerContext, request.NamespacedName.Namespace)
```

Comment:
```suggestion
	r.clusterInfo, _, _, err = opcontroller.LoadClusterInfo( r.context, r.opManagerContext, request.NamespacedName.Namespace,&cephCluster.Spec)
```
generally we keep `context` and `namespace` before other args